indexer/ai/src/main/java/com/google/cloudsearch/ai/AISkillSet.java

```diff
@@ -0,0 +1,66 @@
+package com.google.cloudsearch.ai;
+
+import com.sun.org.apache.bcel.internal.generic.ANEWARRAY;
+import org.checkerframework.checker.units.qual.C;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class AISkillSet {
+
+    private List<AISkill> skillSet;
+    public void setSkillSet(List<AISkill> skillSet){
+
+        this.skillSet = skillSet;
+    }
+    public List<AISkill> getSkillSet(){
+
+        return this.skillSet;
+    }
+    public void parse(JSONObject skillSet){
```

Comment:
this should throw Exception

Reply:
Yes, I am working on the error handling part